src/utils/filterKeyword.js

```diff
@@ -4,5 +4,5 @@
  * @returns string where space is replaced by %20
  */
 module.exports = function filterKeyword(keyword) {
-  return keyword.replaceAll(/\s/g, '%20');
+  return keyword.replace(/\s/g, '%20');
```

Comment:
I have a feeling that this wasn't the only problem caused by the node version as you mention further mis-behaviour. Frustrating that these libraries/technologies/languages are so backward incompatible.